builder/make.sh

```diff
@@ -7,3 +7,25 @@
 
 # Start parallel build
 make -j $(nproc) V=sc world
+
+IMG=$(find /home/build/openwrt/bin -name nextsecurity\*-x86-64-generic-ext4-combined-efi.img.gz);
```

Comment:
Can you please avoid to look for only for x84_64 images?
We could just simply add a loop for all `nextsecurity\*-generic-ext4-combined-efi.img.gz` images